libs/src/sdk/api/ResolveApi.ts

```diff
@@ -1,25 +1,27 @@
 import {
+  JSONApiResponse,
   HTTPHeaders,
-  Playlist,
   RequiredError,
-  Track,
-  User
+  BaseAPI,
+  ResolveRequest
 } from './generated/default'
 import {
-  ResolveApi as GeneratedResolveApi,
-  ResolveRequest
-} from './generated/default/apis/ResolveApi'
+  instanceOfPlaylistResponse,
+  instanceOfTrackResponse,
+  PlaylistResponseFromJSON,
+  TrackResponseFromJSON,
+  UserResponseFromJSON
+} from './generated/default/models'
 
-export class ResolveApi extends GeneratedResolveApi {
+// Extend that new class
+export class ResolveApi extends BaseAPI {
   /**
    * Resolves a provided Audius app URL to the API resource it represents
    */
-  async resolve<T extends Track | Playlist | User>(
-    requestParameters: ResolveRequest
-  ): Promise<T> {
+  async resolveRaw(requestParameters: ResolveRequest) {
```

Comment:
Note that this no longer returns `Track | Playlist | User` but `TrackResponse | PlaylistResponse | UserResponse` and doesn't include a type parameter intentionally (since we don't know what the type is from the input). Consumers will need to cast appropriately.